### PR TITLE
Launchers fix: replace exec with spawn for command

### PR DESCRIPTION
**Pull request by chrmod**
https://github.com/testem/testem/issues/958

### CODE REVIEW

File: lib/process-ctl.js [hunk 1]
```diff
@@ -86,9 +86,17 @@ ProcessCtl.prototype.exec = function(cmd, options) {
   log.info('executing: ' + cmd);
 
   var self = this;
-  this.process = childProcess.exec(cmd, _options, function(err, stdout, stderr) {
-    self.stdout = stdout;
-    self.stderr = stderr;
+  var cmdParts = cmd.split(" ");
```
Comment (johanneswuerbach) [inline lib/process-ctl.js]: This won't support quoted commands like: `node -e "console.log('with spaces');"`

**Comment by chrmod** [comment]
windows issues seems to come from *spawn-args*. @johanneswuerbach any suggestions how to deal with them?

**Comment by johanneswuerbach** [comment]
Looks like `spawn-args` works fine, great find 👏  I think the test can be fixed by replacing this line https://github.com/testem/testem/blob/master/tests/launcher_tests.js#L74 with

`assert.equal(stdout, 'hello\n');`

Not entirely sure why, but the same test using `spawn` is also using `\n` instead of `\r\n` https://github.com/testem/testem/blob/master/tests/launcher_tests.js#L158

**Comment by chrmod** [comment]
@johanneswuerbach all tests green

**Comment by johanneswuerbach** [comment]
Thanks 👍

**Pull request merged**